android/app/build.gradle

```diff
@@ -148,6 +148,7 @@ android {
 dependencies {
     compile project(':react-native-linear-gradient')
     compile project(':magnet-scanner-android')
+    compile project(':reactnativemapboxgl')
```

Comment:
Any reason why we don't use default react-native [`MapView`](https://facebook.github.io/react-native/docs/mapview.html)